garage/tf/samplers/batch_sampler.py

```diff
@@ -41,3 +44,102 @@ def obtain_samples(self, itr):
             paths_truncated = parallel_sampler.truncate_paths(
                 paths, self.algo.batch_size)
             return paths_truncated
+
+    def process_samples(self, itr, paths):
```

Comment:
Does this break VPG or any other TF algo?

Reply:
Does it break anything else?

Reply:
I don't find it break other things.

Reply:
who else uses BatchSampler?

Reply:
BatchPolopt, NPO, PPO, and TRPO. These algorithms work well.

Reply:
i see. okay, please file a bug for VPG and put it on the 2018.08 milestone. it isn't really used much anyway (and it's not a hard fix).